sunpy/net/dataretriever/sources/goes.py

```diff
@@ -199,7 +213,9 @@ def register_values(cls):
         adict = {attrs.Instrument: [
             ("GOES", "The Geostationary Operational Environmental Satellite Program."),
             ("XRS", "GOES X-ray Flux")],
-            attrs.goes.SatelliteNumber: [(str(x), f"GOES Satellite Number {x}") for x in goes_number]}
+            attrs.goes.SatelliteNumber: [(str(x), f"GOES Satellite Number {x}") for x in goes_number], 
+            attrs.goes.VersionData: [('VersionData', ("If `old` then will use the 13, 14, and 15 " 
+                                     "old calibrated data before NOAA re-processed it"))]}
```

Comment:
```python
attrs.goes.VersionData: [('old', "old calibrated XRS data before NOAA re-processed it")]}
```
It should be like this, the adict contains list of tuples of possible values that an attr can take; and its description.
So `old` is one possible value, since we can do `a.goes.VersionData.old` and the second element will explain what old means here.

Reply:
thanks!